src/const/cards.const.ts

```diff
@@ -17,7 +17,8 @@ export const operationalCards: WidgetName[] = [
   WidgetName.injured_count_by_accident_year,
   // WidgetName.accident_count_by_driver_type,
   WidgetName.killed_and_injured_count_per_age_group,
-  WidgetName.accident_count_by_day_night
+  WidgetName.accident_count_by_day_night,
+  WidgetName.killed_and_injured_count_per_age_group_stacked
```

Comment:
minor - but I like to add `,` in the end of the last line as well (and then when the next line wull be added in the future no need to change this one)